python/nano/src/bigdl/nano/pytorch/strategies/ddp_spawn.py

```diff
@@ -221,6 +227,68 @@ def setup(self, trainer: "pl.Trainer") -> None:
             # so we need to set it to `False` manuallay
             self.model.eval()
 
+        if trainer.training and self.auto_lr:
+
+            def _unpack_lightning_optimizer(opt):
+                return opt._optimizer if isinstance(opt, LightningOptimizer) else opt
+
+            optimizers = self.optimizers
+            optimizers = [_unpack_lightning_optimizer(opt) for opt in optimizers]
+
+            for optimizer in optimizers:
+                for param_group in optimizer.param_groups:
+                    param_group["lr"] *= self.world_size
+
+            lr_scheduler_configs = self.lr_scheduler_configs
+            for config in lr_scheduler_configs:
+                scheduler = config.scheduler
+                scheduler.base_lrs = [lr * self.world_size for lr in scheduler.base_lrs]
+
+            if self.lr_scheduler_configs:
+                warnings.warn(f"Nano warmup currently only support none optimizers, "
```

Comment:
I am a little confused about this warning. What do you mean by "none optimizers" and how do you "skip warmup"?

Reply:
"none optimizers" should be "no lr_scheduler", I will fix it.